sdk/monitor/azure-monitor-opentelemetry-autoconfigure/CHANGELOG.md

```diff
@@ -1,12 +1,9 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.2.0 (2025-04-10)
 
 ### Bugs Fixed
+- Fixed a bug causing logs to be instrumented. Verbose logs would lead to recursive logging. ([#44828](https://github.com/Azure/azure-sdk-for-java/pull/44828))
 
```

Comment:
What about bumping the OpenTelemetry SDK version?

Reply:
THAT comment is in the other PR. I'm waiting for the other PR to merge and then i'll deal with any other conflicts.

Reply:
other PR is merged, please re-review when you have a chance